api/subscriptions/views.py

```diff
@@ -40,6 +51,19 @@ def get_queryset(self):
         return self.get_queryset_from_request()
 
 
+class AbstractProviderSubscriptionList(SubscriptionList):
```

Comment:
Make it clear that this is getting the subscription for a given provider for a given user?